cloudinit/config/cc_update_etc_hosts.py

```diff
@@ -88,6 +88,7 @@
         ),
     ],
     "frequency": PER_ALWAYS,
+    "activate_by_schema_keys": [],
```

Comment:
@aciba90 shouldn't this be `manage_etc_hosts`? As I'm reading this code it looks like we default to False if not provided, and a False value drops us into [the final else clause which skips](https://github.com/canonical/cloud-init/blob/main/cloudinit/config/cc_update_etc_hosts.py#L138).

```suggestion
    "activate_by_schema_keys": ["manage_etc_hosts"],
```
One thing to consider if if we do skip altogether, we will not get a debug log like:
```
 Configuration option 'manage_etc_hosts' is not set, not managing /etc/hosts in module update_etc_hosts
```

It's dubious the value of such a log, but I'll let you and @holmanb decide before merging.

Reply:
I have added as in my opinion that log message is quasi-equivalent to the want we emit while skipping modules:

`Skipping modules 'update_etc_hosts, ...' because no applicable config is provided.`

But if @holmanb thinks the opposite, I will revert it.

Reply:
@aciba90 I'm fine with the "skipping .." message in place of "Configuration option...".